std_msgs/Header header
#Range in cm
uint32 range